src/routes/anthropic.ts

```
@@ -0,0 +1,653 @@
+      textContent = textContent
+        .replace(/<tool_call>\s*[\s\S]*?<\/tool_call>/gi, "")
+        .replace(/<tool_call>\s*[\s\S]*$/gi, "") // unclosed trailing block
+        .trim();
+    }
+    if (textContent) {
+      contentBlocks.push({ type: "text", text: textContent });
+    }
+  }
+  if (hasToolCalls) {
+    for (const tc of result.toolCalls!) {
+      let inputObj = {};
```

ESLint

[error] The value assigned to 'inputObj' is not used in subsequent statements.

(no-useless-assignment)